doc/source/rllib/package_ref/algorithm.rst

```diff
@@ -46,8 +63,16 @@ and override one or more of its methods. Those are in particular:
 `See here for an example on how to override Algorithm <https://github.com/ray-project/ray/blob/master/rllib/algorithms/pg/pg.py>`_.
 
 
-Algorithm base class (ray.rllib.algorithms.algorithm.Algorithm)
----------------------------------------------------------------
+.. _rllib-algorithm-api:
+
+Interacting with an Algorithm
+-----------------------------
+
+Once you've built an :py:class:`~ray.rllib.algorithms.algorithm_config.AlgorithmConfig`
+and retrieve an :py:class:`~ray.rllib.algorithms.algorithm.Algorithm`, you can use it
```

Comment:
maybe more clear:
-> ".. and retrieve an algorithm **from it** via the `build()` method" ...

Reply:
yes!